ci/jenkins/utils.sh

```diff
@@ -78,3 +78,19 @@ function switch_golang() {
     echo "====== Switching to Golang ${golang_version} ======"
     ln -s ${GOLANG_RELEASE_DIR}/${golang_version} ${GOLANG_RELEASE_DIR}/go
 }
+
+function docker_login() {
+    set +ex
+    dockerUser=$1
+    dockerPassword=$2
+    for i in `seq 5`; do
+        docker login --username=$dockerUser --password=$dockerPassword
```

Comment:
You may use $1/$2 directly, the options explain the usage clearly.
```suggestion
    for i in `seq 5`; do
        docker login --username=$1 --password=$2
```

Reply:
Updated, thanks.